app/helpers/games_helper.rb

```diff
@@ -1,2 +1,13 @@
 module GamesHelper
+  def render_piece(x, y)
```

Comment:
In a helper, you have access to the variables you set in your controller. Take a look at `games#show`. You should be setting `@game`. So you can use that here.

`@game.pieces` to get the pieces for the current game.

If the game doesn't have any pieces, it's possible this branch is outdated and you need to pull latest from master to get the `populate_board!` method in.